core/src/acc_text/nlg/gf/service.clj

```diff
@@ -41,6 +49,6 @@
                             :body    (json/write-value-as-string request-content write-mapper)})
           {[[_ results]] :results error :error} (json/read-value body read-mapper)]
       (cond
-        (some? request-error) (throw request-error)
-        (some? error) (throw (Exception. ^String error))
-        :else (map #(post-process lang lincat %) (distinct results))))))
+       (some? request-error) (throw request-error)
```

Comment:
changing this line to this:
`(some? request-error) (throw (Exception. (format "GF service host: %s" (.getMessage request-error))))`

we would get an error in console like this:
`2021-02-19 11:13:42,429 ERROR a.n.core - GF service host: Connection refused`
and indication in UI like this:
![Screenshot from 2021-02-19 11-15-21](https://user-images.githubusercontent.com/20424394/108483812-d0029d00-72a3-11eb-80b9-8475a4011f40.png)

Reply:
lets think about this more, "/status" is a useful thing, but maybe we can drop changes to front end and access the endpoint when manually need to check health of services?

Reply:
The point is knowing what is wrong before trying to generate, not generate and fail. For me, I never actually seen this error in UI, it was always no result and had to debug it manually

Reply:
the errors are controlled by `DISPLAY_ERROR` env variable and are not enabled by default (when running api from REPL for example), but they are displayed when running via docker ("make run-app", "make run-dev-env").
and yes, they are definitely more about generation errors, like "dont know how to apply modifier with type X for type Y" and etc., but some indication in UI must exist i think, maybe a red icon on top right or something, as with other UI errors.

Reply:
UI now is showing error when GF service is offline